### DISCUSSION TITLE
Module with "inside" JS script

### DISCUSSION THREAD

**Discussion by bilizz** [breezefront/community#141] [2. Ideas]
Hi, I was searching a way how to use modules, which have JS implemented direct in PHTML files. Why is this done in that way is probbably (as I checked) that part of JS code is generated by PHP such as some tags id.. etc.

How to convince Breeze to leave those JS code inside or how to solve this problem?

**Comment by vovayatsyuk** [comment]
If you'll add the `data-breeze` attribute, breeze will not remove the script:

```js
<script data-breeze>
require([], fn)
</script>
```
